### PR TITLE
Add about page info

### PR DESCRIPTION
**Pull request by DanielN88**
## Description 
This PR adds in an about page. 

#### Where to start?
Start at the about component.

#### How to test?
In the project directory, you can run:

### `npm start`
Runs the app in the development mode.\
Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
The page will reload when you make changes.

#### Screenshots (if appropriate)
Screen Shot 2022-07-18 at 3 36 21 PM

#### Type of Change:
- [] Bug fix (non-breaking change which fixes an issue)
- [] Refactor (code is working and without bugs, formatting has been streamlined)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### How Has This Been Tested?
What steps have you taken to ensure the code is effective?
- [x] Have you used Dev Tools/ ran code in the console?
- [] Have you ran the Cypress testing suite?

#### Checklist:
- [x] My code follows the style guidelines of this project.
- [] My commit messages are consistent, descriptive, concise and begin with a verb and capital letter.
- [x] I have performed a self-review of my own code.
- [x] I have added reviewers.

### CODE REVIEW

**Review by Jlavarine — APPROVED** [review]
Love the new look for the About page

**Comment by aominhlong** [comment]
Love the new about page.

**Review by aominhlong — APPROVED** [review]

File: src/About.js [hunk 1]
```diff
@@ -1,10 +1,61 @@
...
+                    <img src={JoeS} alt='Photo of Joes dog'/>
+                </div>
+                <div className='dev'>
+                    <h3>Jacob L</h3>
+                    <p>Meet their pup Jojo</p>
+                    <img src={JacobL} alt='Photo of Jacobs dog'/>
+                </div>
+                <div className='dev'>
+                    <h3>Josh M</h3>
+                    <p>Meet their pup Pheonix</p>
```
Comment (JoshMallery) [inline src/About.js]: Please change my lil guys name to Phoenix.

**Review by JoshMallery — CHANGES_REQUESTED** [review]
Please make the change of lil phoenix's spelling of his name, the about page looks good. And like you mentioned we can add even more text to the About side.

File: src/About.js (hunk 1 above)
Comment (DanielN88) [reply src/About.js]: Fixed

**Review by JoshMallery — APPROVED** [review]

**Pull request merged**